app/assets/stylesheets/activities.css.scss

```diff
@@ -62,7 +62,7 @@ article {
 
 #base-jumping-card{
     height: 176px;
-    background: url('basebase.jpg') center / cover;
+    background: image-url('basebase.jpg') center / cover;
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Prefer double-quoted strings